src/Assert.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Symfony\Cmf\Component\Testing;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * A class with extra usefull test assertions.
+ *
+ * @author Wouter J <wouter@wouterj.nl>
+ */
+class Assert
+{
+    public static function responseOk(Response $response)
+    {
+        if (200 === $response->getStatusCode()) {
+            return true;
+        }
+
+        // extract exception message from output (assuming the output is the default Symfony exception page)
+        libxml_use_internal_errors(true);
+
+        $dom = new \DomDocument();
+        $dom->loadHTML($response->getContent());
+
+        $xpath = new \DOMXpath($dom);
+        $exceptionElement = $xpath->query('//div[contains(@class,"text-exception")]/h1');
+
+        throw new \LogicException(sprintf(
```

Comment:
IMO, throwing a dedicated exception (AssertionException for instance) would be better than using the very generic LogicException. It makes it easier to catch only assertion failures.